### ISSUE TITLE
Temp file gettint 12 go per day of gw2 cache

### ISSUE THREAD

**Issue by Ryu-Pendragon** [TheCheatsrichter/Gw2_Launchbuddy#452]
I just realise recently that my 50 logging account every day put a damn  12 go of cache in the temp file and im worried about my ssd getting used to fast 

i saw on gw2launcher that there a way to block this by activating a static cache of gw2 but do we have a solution there ?

do launchbuddy is dead btw??? cause im on V2.4.2 till few month cause the main version is broken .

ty in advence and sory for my broken english ( not native language )  :(

**Comment by PremiumQuality** [comment]
Same issue here, ever since updating to the latest free version (3.2.6) I'm also getting 10-20gb of cache files in %temp% folder after logging into alts.

**Comment by AZDarkknight** [comment]
Im getting the same but given its over a month since this was posted with no response, Im guessing its dead.

**Comment by Ryu-Pendragon** [comment]
Yeah man i gived a try to gw2 launcher and it way way better :O . Everyone there should move over to GW2laucher because launchbuddy is completly broken and dead

GW2 launcher have more option and theyre feature work like a charm so overall if you do a lot of dayli like me  ( 50 account ) it way faster with the launcher ...

**Comment by AZDarkknight** [comment]
This isnt stale, its ongoing, but no response from the maker.

**Comment by AZDarkknight** [comment]
Still not stale, just abandoned by the maker who only updates for people that can afford to pay him I guess :(
